src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -81,17 +84,135 @@
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Label Name="FontSize"
+                                   Grid.Row="0"
+                                   Content="Node Font Size" 
+                                   Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                <ComboBox Grid.Row="1"
+                                          Width="213"
+                                          HorizontalAlignment="Left"
+                                          ItemsSource="{Binding FontSizeList}"
+                                          SelectedItem="{Binding Path=SelectedFontSize}"
+                                          Style="{StaticResource NoBordersComboBoxStyle}">
+                                </ComboBox>
+                                
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Run Preview options-->
+                            <Grid Grid.Row="3">
+                                <Grid>
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto"/>
+                                        <ColumnDefinition Width="*"/>
+                                    </Grid.ColumnDefinitions>
+                                    <ToggleButton MinWidth="30"             
+                                                  MaxHeight="20"
+                                                  VerticalAlignment="Center"
+                                                  Grid.Column="0"                              
+                                                  IsChecked="{Binding Path=RunPreviewIsChecked}"
+                                                  Style="{StaticResource EllipseToggleButton1}"/>
+                                    <Label Grid.Column="1"
+                                           Content="Show Run Preview" 
+                                           VerticalAlignment="Center"
+                                           Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                </Grid>
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Number Format options-->
+                            <Grid Grid.Row="4">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Label Grid.Row="0"
+                                       Content="Number Format" 
```

Comment:
Same here

Reply:
@QilongTang 
I found a similar entry in resource files, then I reused the existing one. see the changes in the commit: https://github.com/DynamoDS/Dynamo/pull/11542/commits/9035f3d2b7de82a15cb6098d2a56a00c1b06836d